### PR TITLE
fixed bonding test for back to back

### PR DESCRIPTION
**Pull request by smruti77**
bonding test fixed for back to back.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

File: io/net/bonding.py [hunk 1]
```diff
@@ -355,16 +356,15 @@ def test_run(self):
         self.bond_fail(self.mode)
         self.log.info("Mode %s OK", self.mode)
         self.error_check()
+        time.sleep(5)
```
Comment (narasimhan-v) [inline io/net/bonding.py]: @smruti77 Please document reason for sleep here also

**Comment by narasimhan-v** [comment]
@smruti77 please upload job.log

**Comment by smruti77** [comment]
[bonding.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4212983/bonding.log)

File: io/net/bonding.py [hunk 2]
```diff
@@ -186,7 +185,7 @@ def bond_remove(self, arg1):
             cmd += 'rmmod bonding;'
             cmd += 'ip addr add %s/%s dev %s;ip link set %s up;sleep 5;'\
                    % (self.peer_first_ipinterface, self.net_mask[0],
-                      self.peer_first_interface, self.peer_first_interface)
+                      self.peer_interfaces[0], self.peer_interfaces[0])
```
Comment (narasimhan-v) [inline io/net/bonding.py]: @smruti77 Why not `self.peer_first_interface` ?

Comment (smruti77) [reply io/net/bonding.py]: this interface is getting from peer_ip and that is assign to to bonding interface, at the time of next test the interface is coming as "bondtest" so couldn't assign ip to that interface and peer connection aslo failing.

Comment (narasimhan-v) [reply io/net/bonding.py]: ah ok.

**Pull request merged**